wdl/src/cli.rs

```diff
@@ -152,22 +157,26 @@ pub fn parse_inputs(
             bail!("document does not contain a task or workflow named `{name}`");
         }
     } else {
-        // Neither a inputs file or name was provided, look for a single task or
-        // workflow in the document
-        let mut iter = document.tasks();
-        let (name, inputs) = iter
-            .next()
-            .map(|t| (t.name().to_string(), Inputs::Task(Default::default())))
-            .or_else(|| {
-                document
-                    .workflow()
-                    .map(|w| (w.name().to_string(), Inputs::Workflow(Default::default())))
-            })
-            .context("inputs file is empty and the WDL document contains no tasks or workflow")?;
-
-        if iter.next().is_some() {
-            bail!("inputs file is empty and the WDL document contains more than one task");
-        }
+        // Neither an inputs file or name was provided, look for a workflow in the
+        // document; failing that, find at most one task in the document
```

Comment:
Do we really need to support this use case? I feel like if you are providing neither a name nor an inputs file, it's fine for us to just fail. I guess there isn't any harm in supporting this case, it just strikes me as a bit odd.

Reply:
It's useful for when there's a workflow you want to run that either has no inputs or all inputs have default values, so one doesn't need to specify an inputs file or explicitly give it a name.